compiler_opt/tools/extract_ir.py

```diff
@@ -79,6 +79,12 @@
     'The section name passed to llvm-objcopy. For ELF object files, the '
     'default .llvmbc is correct. For Mach-O object files, one should use '
     '__LLVM,__bitcode')
+flags.DEFINE_enum(
```

Comment:
doesn't absl already have a 'verbosity' flag?

Reply:
Ah, yep. It doesn't seem to be documented outside of https://github.com/abseil/abseil-py/blob/9e543208a72300a4f8677fe725550fe8dc242bac/absl/logging/__init__.py#L165 (and of course the logging documentation page doesn't seem to exist on Github, so I can't open a PR against it), so I've added some information on usage in the docstring. Thanks for pointing this out!